### ISSUE TITLE
How to finetune the RAM++ using object detection dataset without image caption data

### ISSUE THREAD

**Issue by jwwangchn** [xinyu1205/recognize-anything#122]
I have an object detection dataset that only contains bbox and class annotations. How can I use this dataset to train RAM++?

**Comment by SANJINGSHOU14** [comment]
maybe u can use [LLaVa](https://github.com/haotian-liu/LLaVA) to get the caption of the image in your object detection dataset first, then set the class annotations as the union_tag and use them to generate the tag description, after that u have all the needed information.

**Comment by xinyu1205** [comment]
The inference of LLaVA is time-consuming and may generate hallucination captions.
Compared to LLaVA, I recommend attempting [Tag2Text](https://arxiv.org/abs/2303.05657), which is both efficient and detailed captions.

Actually, a simple method is you can directly finetune RAM++ only on image tagging (without image-text alignment) using your class annotations.

**Comment by jwwangchn** [comment]
@xinyu1205 Thank you for your reply. Directly finetuning the image tagging task is exactly what I want. Should I directly set the loss of the Generation branch to 0? Can I train my dataset by simply running the `finetune.py` file?

**Comment by xinyu1205** [comment]
Hi, the [RAM_plus](https://github.com/xinyu1205/recognize-anything/blob/main/ram/models/ram_plus.py) model have image tagging branch and image-text alignment branch (all with in a shared tagging_head), you can only using the image tagging branch.
